README.md

```diff
@@ -53,8 +53,8 @@ During the first boot, there are some initial configurations necessary, which wi
 
 
 ### OMXPlayer
-
-To build the customized OMXPlayer, run the command below:
+We have build our own customized OMXPlayer and make a [v1.0.0 release](https://github.com/KaneBetter/omxplayer/releases/tag/v1.0.0) in this [repo](https://github.com/KaneBetter/omxplayer).
```

Comment:
```suggestion
The PIP team has built a customized version of OMXPlayer and made a dedicated [release](https://github.com/KaneBetter/omxplayer/releases/tag/v1.0.0) in this [repo](https://github.com/KaneBetter/omxplayer).
```

As mentioned in #57 the readme deserves an acknowledge section to clarify and credit the PIP team/context.

Reply:
Done